client/components/action-card/test/index.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * @jest-environment jsdom
+ */
+import '@testing-library/jest-dom';
+import { shallow } from 'enzyme';
+import { axe, toHaveNoViolations } from 'jest-axe';
+expect.extend( toHaveNoViolations );
```

Comment:
This is the way it was listed in the examples I had from other people using the jest-axe package. I think it's to give clearer context as to why expect is being extended. I'm pretty neutral on having it there though. It could just as easily go below the import block.